docs/release_notes/v1.7.1.md

```diff
@@ -26,3 +27,17 @@ A change introduced in release 1.6 caused Timers to be invalidated if their `Due
 #### Solution
 
 Revert the code to pre-1.6 behavior which allows `DueTimes` to be in the past as that still equates to firing immediately.
+
+### Panic: nil pointer dereference in API Logging
+
+#### Problem
+
+When enabling API logging, a nil reference can happen and crash the sidecar.
+
+#### Root cause
+
+Logic for API logging in gRPC assumed gRPC `info` object was always not nil, but user reported it could be nil when invoking an Actor as per issue [#4527](https://github.com/dapr/dapr/issues/4527).
```

Comment:
```suggestion
Logic for API logging in gRPC assumed gRPC `info` object was always not nil, however it has been reported that it could be nil when invoking an actor as per issue [#4527](https://github.com/dapr/dapr/issues/4527).
```